src/components/Tab/styled/button.js

```diff
@@ -43,30 +44,42 @@ const StyledAnchor = attachThemeAttrs(styled.button)`
     }
 
     &:hover {
-        background-color: ${props => props.palette.action.hover};
         color: ${props => props.palette.text.label};
         z-index: 1;
     }
 
     &:focus {
         text-decoration: none;
         outline: 0;
-        border-radius: 14px 14px 0 0;
     }
 
-    &::after {
-        content: '';
-        position: absolute;
-        left: -2px;
-        height: 20px;
-        width: 1px;
-        background-color: ${props => props.palette.border.divider};
-        box-sizing: border-box;
-    }
+    ${props =>
+        (props.variant === 'card' || !variants.includes(props.variant)) &&
```

Comment:
there is not much implication since there are only two elements in the array but here you can use a map instead, thus avoiding the iteration